sdcm/teardown_validators/events.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import re
+
+from sdcm.sct_events import Severity
+from sdcm.sct_events.file_logger import get_events_grouped_by_category
+from sdcm.teardown_validators.base import TeardownValidator
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ErrorEventsValidator(TeardownValidator):  # pylint: disable=too-few-public-methods
```

Comment:
We should document the validator somewhere

A docstring on the class should be a good start

Later some doc under docs/ would be also helpful, to point to those

Reply:
added